__tests__/actionCreators/resources.test.js

```diff
@@ -50,6 +50,16 @@ const mockStore = configureMockStore([thunk])
 // This removes circular references.
 const safeAction = (action) => JSON.parse(JSON.safeStringify(action))
 
+const cloneAddResourceActionAsNewResource = (addResourceAction) => {
+  const clonedAction = _.cloneDeep(addResourceAction)
+
+  clonedAction.payload.uri = null
+  clonedAction.payload.group = null
+  clonedAction.payload.editGroups = []
+
+  return clonedAction
+}
```

Comment:
@justinlittman Since this is used in two places, I decided to pull it into its own function so it has a name. I don't necessarily love the name, so pinging you for explicit review of this piece.